src/globals.ts

```diff
@@ -50,7 +50,7 @@ export function installGlobals() {
   global.File = NodeFile
 
   global.Headers = NodeHeaders as typeof Headers
-  global.Request = NodeRequest as typeof Request
```

Comment:
```sh
FAIL test/server.test.ts
  ● Test suite failed to run

    src/globals.ts:53:20 - error TS2352: Conversion of type 'typeof NodeRequest' to type '{ new (input: RequestInfo | URL, init?: RequestInit | undefined): Request<string>; prototype: Request<string>; }' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
      Types of property 'prototype' are incompatible.
        Type 'NodeRequest' is missing the following properties from type 'Request<string>': param, query, queries, header, and 2 more.

    53   global.Request = NodeRequest as typeof Request
                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Test Suites: 1 failed, 1 total
```


_...convert the expression to 'unknown' first._